end-to-end-tests/src/test/java/io/eventuate/examples/tram/ordersandcustomers/endtoendtests/CustomersAndOrdersEndToEndTest.java

```diff
@@ -36,9 +30,17 @@
   @Value("${host.name}")
   private String hostName;
 
+  private String baseUrlOrdersProxy(String... path) {
```

Comment:
All HTTP requests should go through the API Gateway - that's its purpose.

Reply:
ok